hilla/src/main/templates/dev/hilla/shrinkwrap/VaadinCoreShrinkWrap.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package dev.hilla.shrinkwrap;
+
+import com.vaadin.flow.component.dependency.NpmPackage;
+
+/**
+ * Empty class which makes flow add npm vaadin-core-shrinkwrap dependency.
+ */
+@NpmPackage(value = "@vaadin/vaadin-core-shrinkwrap", version = "${shrink.version}")
```

Comment:
I think we can add here `@NpmPackage(value="@vaadin/vaadin-core")` and package.json file will be updated with all the components.

Reply:
Per discussion, removed this file all together